frontend/toolbar/main_toolbar_mobile.html

```diff
@@ -0,0 +1,65 @@
+<md-toolbar md-colors="{background: 'teal-500'}" 
+    layout-align="center" layout="row" ng-if="mainToolbarCtrl.isMobileScreen()" style="overflow: hidden;">
```

Comment:
this ng-if makes sense here? 
the place that uses the component should know when it has to appears.

Reply:
I chose to put it there because it would live only there, do u wanna me to change that?